specs/eventing/control-plane.md

```diff
@@ -0,0 +1,985 @@
+replies to the Broker's event-delivery input and instead use an internal
+queueing mechanism.
+
+### Topology Based Routing
+
+A Channel MUST publish a URL at `status.address.url` when it is able to receive
+events. This URL MUST accept CloudEvents in both the
+[Binary Content Mode](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#31-binary-content-mode)
+and
+[Structured Content Mode](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#32-structured-content-mode)
+HTTP formats. Before sending an HTTP response, the Channel MUST durably enqueue
+the event (be able to deliver with retry without receiving the event again).
+
+For each event received by the Channel, the Channel MUST deliver the event to
+each associated Subscription **at least once** (where "associated" means a
+Subscription with a `spec.channel` which references the Channel). If the
+Subscription has a `Ready` condition of `true` when the event is evaluated, the
+Channel MUST forward the event as described in
+[event delivery as described below](#event-delivery). The Channel MAY also
+forward events to associated Subscriptions for with the `Ready` condition is not
+currently `true`. (One example: a Subscription which is in the process of being
+programmed in the Channel data plane might receive _some_ events before the data
+plane programming was complete and the Subscription was updated to set the
+`Ready` condition to `true`.)
+
+If multiple Subscriptions with the same destination are associated with the same
+Channel, each Subscription MUST generate one delivery attempt per Subscription.
+The implementation MAY attach additional event attributes or other metadata
+distinguishing between these deliveries. The implementation MUST NOT modify the
+event payload in this process.
+
+### Event Delivery
+
+Once a Trigger or Subscription has decided to deliver an event, it MUST do the
+following:
+
+1. Resolve all URLs and delivery options, using the values in `status` for URL
+   resolution.
+
+1. Attempt delivery to the `status.subscriberUri` URL following the
+   [data plane contract](./data-plane.md).
+
+   1. If the event delivery fails with a retryable error, it SHOULD be retried
+      up to `retry` times, following the `backoffPolicy` and `backoffDelay`
+      parameters if specified.
+
+1. If the delivery attempt is successful (either the original request or a
+   retry) and no event is returned, the event delivery is complete.
+
+1. If the delivery attempt is successful (either the original request or a
+   retry) and an event is returned in the reply, the reply event will be
+   delivered to the `status.replyUri` destination (for Subscriptions) or added
+   to the Broker for processing (for Triggers). If `status.replyUri` is not
+   present in the Subscription, the reply event is be dropped.
+
+   1. If delivery of the reply event fails with a retryable error, the delivery
```

Comment:
it might be too to make it clear that this Sub-bullet is just for Subscriptions, not Triggers